chimney/src/test/scala/io/scalaland/chimney/PBTransformationSpec.scala

```diff
@@ -51,6 +52,15 @@ object PBTransformationSpec extends TestSuite {
         pb.addressbook.PhoneType.WORK
     }
 
+    "handle transformation of proto with 'Unrecognized' instance properly" - {
+      (pb.addressbook.PhoneType.HOME: pb.addressbook.PhoneType).transformInto[addressbook.PhoneType] ==> addressbook.HOME
+
+      val ex = intercept[EnumUnrecognizedInstanceException](
+        (pb.addressbook.PhoneType.Unrecognized(1): pb.addressbook.PhoneType).transformInto[addressbook.PhoneType]
+      )
+      ex.getMessage ==> "mapping for 'Unrecognized' proto enum instance is abscent in io.scalaland.chimney.examples.addressbook.PhoneType subtypes"
```

Comment:
I think it's better to test `sourceTypeName` and `targetTypeName` instead of message